src/ontology/mondo.Makefile

```diff
@@ -624,6 +629,12 @@ open_%_report:
 mondo_obo:
 	robot convert -i mondo-edit.obo -f obo -o mondo-edit.obo
 
+tmp/mondo-ingest.owl:
+	curl https://github.com/monarch-initiative/mondo-ingest/releases/latest/download/mondo-ingest.owl -L --output $@
```

Comment:
I was worried that this wouldn't be updated, but it looks like Nico has been doing monthly updates at least.